pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -610,6 +610,11 @@ public void start() throws PulsarServerException {
                 throw new IllegalArgumentException("brokerServicePort/brokerServicePortTls must be present");
             }
 
+            if (config.isTlsEnabled()
+                    && (!config.getWebServicePortTls().isPresent() || !config.getBrokerServicePortTls().isPresent())){
+                throw new IllegalArgumentException("webServicePortTls/brokerServicePortTls must be present");
```

Comment:
nit: the error message can be more informative like `webServicePortTls/brokerServicePortTls must be set when TlsEnabled set to true`

Reply:
It is better to be consistent with the current state of the project,  same of `brokerServicePort/brokerServicePortTls must be present`It is better to be consistent with the current state of the project,  same of `brokerServicePort/brokerServicePortTls must be present`